synapse/cortex.py

```diff
@@ -597,7 +597,8 @@ def _initPushLoop(self):
                             continue
 
                         size = len(items)
-                        indx = self.layer.splicelog.indx
+                        indx = (await self.layer.stat())['splicelog_indx']
+
```

Comment:
while we are near this code, can we change the logging statement below to

`logger.info('splice push: %d %d/%d (%.4f%%)', size, offs, indx, perc)`

?

Reply:
done

Reply:
(in another branch)